rector.php

```diff
@@ -27,7 +27,7 @@
     $parameters->set(Option::SETS, [
         SetList::NAMING,
         SetList::CODE_QUALITY,
-        SetList::REPOSITORY_AS_SERVICE,
+        SetList::DOCTRINE_REPOSITORY_AS_SERVICE,
```

Comment:
I think the const should change into `DOCTRINE_REPOSITORY_AS_SERVICE` because the `REPOSITORY_AS_SERVICE` const is undefined.